core/templates/domain/utilities/FileDownloadRequestObjectFactory.ts

```diff
@@ -38,14 +33,12 @@ export class FileDownloadRequest {
   providedIn: 'root'
 })
 export class FileDownloadRequestObjectFactory {
-  // TODO(#9996): Replace 'any' with the exact type. This has kept as
-  // 'any' since canceler is a 'Deferred' type object which is native to
-  // AngularJS and does not have a type in native typescript.
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Can you add a lint check that disallows "no-explicit-any" anywhere in the codebase? Probably just adding it to the list of forbidden strings suffices (but make sure it applies to the right files).

In general, think about how to ensure that the measures you are putting in place now will hold forever.

Reply:
done